decidim-admin/app/packs/src/decidim/admin/choose_language.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-invalid-this */
+/* eslint-disable require-jsdoc */
 
-$(() => {
+export default function initLanguageChangeSelect() {
```

Comment:
I think it would be better if we provided the elements for which the language selector is initiated to the method, as in:
```js
export default function initLanguageChangeSelect(elements) {
  elements.forEach((select) => {
    // ...
  });
}
```

And then when initiating the element, do something like:
```js
window.addEventListener("DOMContentLoaded", () => {
  initLanguageChangeSelect(document.querySelectorAll("select.language-change"));
});
```

This way we would not keep adding multiple event listeners for each of the elements when the new records are added at for the forms.

Reply:
Done. Thanks for the explanation!